### PR TITLE
Improve .editorconfig

### PR DESCRIPTION
**Pull request by evgfilim1**
This is a continuation of https://github.com/aiogram/aiogram/pull/653#pullrequestreview-721944497

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -3,21 +3,16 @@ root = true
 [*]
```
Comment (evgfilim1) [inline .editorconfig:3]: As spec says, `[*]` won't match every file in all subdirectories, but `[**]` will.

image

File: .editorconfig [hunk 2]
```diff
@@ -1,23 +1,18 @@
 root = true
 
-[*]
+[**]
```
Comment (Olegt0rr) [inline .editorconfig:3]: Why **?

**Review by Olegt0rr — APPROVED** [review]

File: .editorconfig [hunk 3]
```diff
@@ -1,23 +1,18 @@
...
 trim_trailing_whitespace = true
-
-[*.py]
-indent_style = space
-indent_size = 4
 max_line_length = 99
 
-[*.{yml, yaml}]
-indent_style = space
+[*.{yml,yaml}]
```
Comment (Olegt0rr) [inline .editorconfig:12]: [**.{yml,yaml}]

File: .editorconfig [hunk 4]
```diff
@@ -1,23 +1,18 @@
...
 indent_size = 2
 
 [*.{md,txt}]
```
Comment (Olegt0rr) [inline .editorconfig:15]: `[**.{md,txt,rst}]`

**Review by Olegt0rr — APPROVED** [review]

**Review by JrooTJunior — APPROVED** [review]

**Pull request merged**